clang/include/clang/Basic/BuiltinsNVPTX.def

```diff
@@ -159,6 +159,7 @@ BUILTIN(__nvvm_read_ptx_sreg_pm3, "i", "n")
 
 BUILTIN(__nvvm_prmt, "UiUiUiUi", "")
 BUILTIN(__nvvm_exit, "v", "r")
+BUILTIN(__nvvm_reflect, "UicC*", "r")
```

Comment:
Now that we're exposing it to the end users. We should probably document what it does.
Probably somewhere in https://clang.llvm.org/docs/LanguageExtensions.html

Reply:
It's in the `NVPTXUsage.rst`, and https://github.com/llvm/llvm-project/pull/81253 mentions it more directly.